JavaScript/JavaScriptSDK/AppInsights.ts

```diff
@@ -104,10 +104,15 @@ module Microsoft.ApplicationInsights {
             }
 
             // enable session storage buffer experiment
-            this.config.enableSessionStorageBuffer = new SplitTest().isEnabled(this.config.instrumentationKey, 10); ;
+            var enableExperiment = new SplitTest().isEnabled(this.config.instrumentationKey, 10);
+            this.config.enableSessionStorageBuffer = enableExperiment;
 
             this.context = new ApplicationInsights.TelemetryContext(configGetters);
 
+            DataLossAnalyzer.appInsights = this;
```

Comment:
DataLossAnalyzer.ts :unamused: 

The class was created a while ago and we used it before to measure the data loss rate. I need to enable it for a couple days to measure if session storage buffer is working as expected.